tests/ingresstests/l7_rest_test.go

```diff
@@ -829,6 +829,7 @@ func TestMultiHostIngressStatusCheck(t *testing.T) {
 	}
 	integrationtest.AddSecret("my-secret-v2", "default", "tlsCert", "tlsKey")
 	integrationtest.AddSecret("my-secret", "default", "tlsCert", "tlsKey")
+	time.Sleep(2 * time.Second) //secrets should be created before creating ingress
```

Comment:
@arihantg: Instead of adding sleep, if you think, secret creation is taking time, should we add it as part of `SetUpTestForIngress` function? so that secret will be created in sufficient time.

Reply:
this secret is used only in this UT so can;t be added in SetUpTestForIngress

Reply:
instead of adding the sleep, we can check where the secret is present in the shared informer cache first and then proceed with the ingress creation.